packages/cli/src/commands/build/buildFrontend.ts

```diff
@@ -19,15 +19,17 @@ import { resolve as resolvePath } from 'path';
 import { buildBundle } from '../../lib/bundler';
 import { getEnvironmentParallelism } from '../../lib/parallel';
 import { loadCliConfig } from '../../lib/config';
+import { CliConfigOptions } from '@backstage/config-loader/src/lib/cli';
```

Comment:
sub-path imports aren't allowed, everything needs to be exported from the top level